src/style.css

```diff
@@ -0,0 +1,134 @@
+@import"./style-stars.css";
+
+.catalog-page {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  --black: #060b35;
+  --blue: #00acdc;
+  --grey-white: #f3f3f3;
+  --grey: #616070;
+  --white: #fff;
+}
+
+.body {
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
remove it everywhere except the html seelctor

Reply:
If I remove 'border-box' from everywhere except html tag, it doesn't pass the npm test. It works only if border-box relates to each block containing anything inside it. 
Attaching screens further:
![image](https://user-images.githubusercontent.com/121205546/213898374-1e97480c-8b4a-409a-be96-7131bf99495a.png)
![image](https://user-images.githubusercontent.com/121205546/213898383-66665686-3757-4cd5-8bde-e78167e5b311.png)